submit-web/src/components/SubmissionItem/ManagementPlanSubmission/ManagementPlanProponentView/FormFieldSection.tsx

```diff
@@ -1,23 +1,46 @@
 import ControlledRadioGroup from "@/components/Shared/controlled/ControlledRadioGroup";
 import { YesNoRadioOptions } from "@/components/Shared/YesNoRadioOptions";
-import { Divider, Grid, Typography } from "@mui/material";
+import { Grid, Typography } from "@mui/material";
 import { BCDesignTokens } from "epic.theme";
 import { ManagementPlanSubmissionForm } from "../ManagementPlanStaffView";
-import { FieldErrors } from "react-hook-form";
+import { FieldErrors, get } from "react-hook-form";
 import ControlledTextField from "@/components/Shared/controlled/ControlledTextField";
-
+import { useParams } from "@tanstack/react-router";
+import { useGetSubmissionPackage } from "@/hooks/api/usePackages";
+import { useMemo } from "react";
+import { BarBlueTitle } from "@/components/Shared/Text/BarTitle";
 interface FormFieldSectionProps {
   errors: FieldErrors<ManagementPlanSubmissionForm>; // Replace FormValues with your actual form schema interface
 }
 
 export default function FormFieldSection({ errors }: FormFieldSectionProps) {
+  const {
+    projectId: accountProjectIdParam,
+    submissionPackageId: submissionPackageIdParam,
+  } = useParams({
+    from: "/proponent/_proponentLayout/projects/$projectId/_projectLayout/submission-packages/$submissionPackageId/_submissionLayout/submissions/$submissionId",
+  });
+  const accountProjectId = Number(accountProjectIdParam);
+  const submissionPackageId = Number(submissionPackageIdParam);
+  const { data: submissionPackage } = useGetSubmissionPackage({
+    packageId: submissionPackageId,
+    enabled: Boolean(accountProjectId),
+  });
+
+  const condition = useMemo(() => {
+    if (!submissionPackage?.meta) return "";
+    const condition = get(submissionPackage, "meta.main_condition");
+
+    return get(condition, "condition_number", "");
+  }, [submissionPackage]);
+
   return (
     <>
       <Grid item xs={12} container>
         <Grid item xs={12}>
           <Typography variant="body1">
-            Does the plan address all the requirements in the (condition
-            number)?
+            Does the plan address all the requirements in condition
+            {condition}?
```

Comment:
might need a space here between condition and number

Reply:
Thank you! I missed that.